lib/matplotlib/figure.py

```diff
@@ -3168,8 +3168,8 @@ def __setstate__(self, state):
         if version != mpl.__version__:
             _api.warn_external(
                 f"This figure was saved with matplotlib version {version} and "
-                f"is unlikely to function correctly.")
-
+                f"loaded with {mpl.__version__} so is unlikely to function correctly."
```

Comment:
Take or leave: I find

```suggestion
                f"loaded with {mpl.__version__} so may not function correctly."
```

a bit more neutral. I think we have quite some internal stability and don't expect breaking changes with every version update.